### PR TITLE
Add the installation script for nagios plugins

### PR DESCRIPTION
**Pull request by youngklee**

### CODE REVIEW

File: bin/nagios-plugins.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env bash
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
```
Comment (ed-) [inline bin/nagios-plugins.sh]: 2017

File: bin/nagios-plugins.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        return $RC
+    fi
+
+    DEST=/root/rpm/nagios
+    if [[ ! -z "$1" ]]; then
+        DEST=$1
+        shift
+    fi
+    pushd $DEST
+    yum localinstall -y $COMMON_NAME-$COMMON_VER-$PKG_SUFFIX
```
Comment (ed-) [inline bin/nagios-plugins.sh]: catch the RC of this line like L78-82

File: bin/nagios-plugins.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    done
+    popd
+    return 0
+}
+
+if [[ "$1" == "install" ]]; then
+    shift
+    install $@
+elif [[ "$1" == "download" ]]; then
+	shift
```
Comment (ed-) [inline bin/nagios-plugins.sh]: this is a nitpick but the indentation is inconsistent here

File: bin/nagios-plugins.sh [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  	download $@
+else
+    help
+fi
+return $?
```
Comment (ed-) [inline bin/nagios-plugins.sh]: will this return the error codes caught as RC, or will it be 0 as long as `fi` succeeds?
besides that you can't return unless you're in a function, this should be exit

File: bin/nagios-plugins.sh (hunk 1 above)
Comment (youngklee) [reply bin/nagios-plugins.sh]: Revised as suggested.

File: bin/nagios-plugins.sh (hunk 2 above)
Comment (youngklee) [reply bin/nagios-plugins.sh]: Revised as suggested.

File: bin/nagios-plugins.sh (hunk 3 above)
Comment (youngklee) [reply bin/nagios-plugins.sh]: Revised as suggested.

File: bin/nagios-plugins.sh (hunk 4 above)
Comment (youngklee) [reply bin/nagios-plugins.sh]: Fixed as suggested.

**Comment by youngklee** [comment]
The management has decided to include the nagios plugins in the image for Thebe. This script is no longer needed.

**Pull request closed without merging**